src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -285,7 +285,11 @@ private static void ItemEqual(String[] actualList, List<FileData> expectedList,
                     {
                         String entryName = i.FullName;
 
-                        archive.CreateEntry(entryName.Replace('\\', '/') + "/");
+                        ZipArchiveEntry e = archive.CreateEntry(entryName.Replace('\\', '/') + "/");
+                        try
+                        { e.LastWriteTime = i.LastModifiedDate; }
+                        catch (ArgumentOutOfRangeException)
```

Comment:
I don't see how it could be. We explicitly store the LastModifiedData in the `FileData` class and none of them are invalid. I took this from the logic below for File; I'll remove it in both places.